option.go

```diff
@@ -18,6 +18,19 @@ func File(name string) Option {
 	}
 }
 
+// NoFile returns an option which disables any file lookup
+//
+// This option effectively renders any `File` and `Dir` options useless. This option
+// is most useful in conjunction with the `UseEnv` option when you want to provide
+// config values only via environment variables.
+//
+//   fig.Load(&cfg, fig.NoFile(), fig.UseEnv("my_app"))
+func NoFile() Option {
```

Comment:
I believe `IgnoreFile` would be a more descriptive name here, since we're telling `fig` to actually ignore the file (even if it exists).